src/game/common/compression/compressionmanager.cpp

```diff
@@ -89,6 +89,51 @@ uint32_t CompressionManager::Get_Compression_FourCC(CompressionType type)
     }
 }
 
+/**
+ * @brief Thyme specific: Get the FourCC for the corresponding compression type
+ */
+CompressionType CompressionManager::Get_Compression_Type_By_FourCC(uint32_t fourcc)
+{
+    switch (fourcc) {
+        case FourCC<'E', 'A', 'R', '\0'>::value:
+            return COMPRESSION_EAR;
+        case FourCC<'Z', 'L', '1', '\0'>::value:
+            return COMPRESSION_ZL1;
+        case FourCC<'Z', 'L', '2', '\0'>::value:
+            return COMPRESSION_ZL2;
+        case FourCC<'Z', 'L', '3', '\0'>::value:
+            return COMPRESSION_ZL3;
+        case FourCC<'Z', 'L', '4', '\0'>::value:
+            return COMPRESSION_ZL4;
+        case FourCC<'Z', 'L', '5', '\0'>::value:
+            return COMPRESSION_ZL5;
+        case FourCC<'Z', 'L', '6', '\0'>::value:
+            return COMPRESSION_ZL6;
+        case FourCC<'Z', 'L', '7', '\0'>::value:
+            return COMPRESSION_ZL7;
+        case FourCC<'Z', 'L', '8', '\0'>::value:
+            return COMPRESSION_ZL8;
+        case FourCC<'Z', 'L', '9', '\0'>::value:
+            return COMPRESSION_ZL9;
+        default:
+            captainslog_error("Compression fourcc '%i' unhandled", fourcc);
+            return COMPRESSION_NONE;
+    }
+}
+
+#ifndef GAME_DLL
```

Comment:
Minor: Prefer non negated macros.

Aka favor `#ifdef GAME_DLL` over `#ifndef GAME_DLL`.

This way there are less mental gymnastics required when looking at conditions.